test/modelrepository.js

```diff
@@ -142,7 +142,7 @@ contract('ModelRepository - Evaluating Gradients', accounts => {
     const balanceAfterEvaluation = await balanceOf(patTheGradientProvider)
 
     assert.ok(balanceAfterEvaluation.valueOf() > balanceBeforeEvaluation.valueOf(),
-      'gradient provider was paid')
+      'gradient provider was not paid')
```

Comment:
why not paid? `balanceAfterEvaluation.valueOf() > balanceBeforeEvaluation.valueOf()` seems to state the provider was paid.

Reply:
Yes the provider is paid but shouldn't the 'message'  should be specifying the error message from the assertion error ? In this case, if  'GradientProvider' balance not increase then errorMessage be "GradientProvider not paid' ?

Reply:
I mean Message is shown only when the assert()  fails right?

Reply:
true, my bad